tool/scorecard.dart

```diff
@@ -142,7 +142,7 @@ class LintScore {
       switch (detail) {
         case Detail.rule:
           sb.write(
-              ' [$name](https://dart-lang.github.io/linter/lints/$name.html) |');
+              ' [$name](https://dart.dev/tools/linter-rules/$name.html) |');
```

Comment:
Consider this redirect for increased flexibility:

```suggestion
              ' [$name](https://dart.dev/lints/$name) |');
```

Reply:
Side note, is this scorecard tooling used for anything currently?